qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -130,7 +129,6 @@ def setUp(self):
 
         if self.REQUIRE_FILESYSTEM:
             self.fs = self.mds_cluster.newfs(create=True)
-            self.fs.mds_restart()
```

Comment:
Looks good, but is [this](https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/cephfs_test_case.py#L141) necessary now?

Reply:
The OSD caps? Why wouldn't it be necessary?

Reply:
Sorry wrong url. This is the correct one: https://github.com/ceph/ceph/blob/b8bfa9f6656ea6dcc8404c532113895f57a54d7c/qa/tasks/cephfs/cephfs_test_case.py#L141

Reply:
Yes, we still need that actually. The check is really waiting for all the ranks to be active. I'll fix the comment.